src/gcp_channel_factory.ts

```diff
@@ -80,17 +81,23 @@ export function getGcpChannelFactoryClass(
           'Channel options must be an object with string keys and integer or string values'
         );
       }
+      this.minSize = 1;
       this.maxSize = 10;
       this.maxConcurrentStreamsLowWatermark = 100;
       const gcpApiConfig = options.gcpApiConfig;
       if (gcpApiConfig) {
         if (gcpApiConfig.channelPool) {
           const channelPool = gcpApiConfig.channelPool;
+          if (channelPool.minSize) this.minSize = channelPool.minSize;
```

Comment:
By default the min size will be 0 because of the protobuf defaults. I think we should check for that if we always want to start with at least 1 channel.

Also, we might need a test to verify that min number of channels are created after construction.